src/gdgem/tests/HitGenerator.cpp

```diff
@@ -13,7 +13,7 @@ namespace Gem {
 //
 void HitGenerator::printHits() {
   for (auto & Hit : Hits) {
-    fmt::print("t {}, p {}, c {}, w {}\n", Hit.time, Hit.plane, Hit.coordinate, Hit.weight);
+    fmt::print("t {}, p {}, c {}, w {}\n", uint64_t(Hit.time), int(Hit.plane), int(Hit.coordinate), int(Hit.weight));
```

Comment:
I thought the purpose/advantage of fmt::* was that it took care of this automatically?

Reply:
It does, the issue is with GCC and the use of packed structs, not with FMT.